src/main/java/core/basesyntax/MyCalculator.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+public class MyCalculator implements Calculator {
+    public double calculate(double a, double b, char sing) throws MyCalculateException {
+        switch (sing) {
+            case '*': {
```

Comment:
```suggestion
            case '*': {
```
I advise using Constant

Reply:
I did it.